truffle/src/main/java/org/jruby/truffle/nodes/ext/BigDecimalNodes.java

```diff
@@ -354,126 +545,144 @@ public Object addSpecial(RubyBasicObject a, RubyBasicObject b) {
             else
                 return b;
         }
-
-        // TODO (pitr 28-may-2015): should it support coerce?
     }
 
-    @CoreMethod(names = "add", required = 2)
-    public abstract static class AddNode extends BigDecimalCoreMethodNode {
+    @CoreMethod(names = {"+"}, required = 1)
```

Comment:
Unfortunately, I don't think you were around when we discussed some code style stuff.  If there's only one method name, we've opted to omit the braces.  If the braces are used, the strings are space padded (`{ "+" }`).